db/openshift.deploy.yml

```diff
@@ -326,3 +329,86 @@ objects:
               dnsPolicy: "ClusterFirst"
               serviceAccountName: "${JOB_SERVICE_ACCOUNT}"
               serviceAccount: "${JOB_SERVICE_ACCOUNT}"
+  - kind: CronJob
+    apiVersion: "batch/v1"
+    metadata:
+      name: ${NAME}-${ZONE}-${COMPONENT}-restore
+      labels:
+        app: ${NAME}-${ZONE}
+        cronjob: ${NAME}-${ZONE}
+    spec:
+      schedule: "0 0 31 2 *" # At 00:00 on day-of-month 31 in February, so it won't run, we'll trigger it manually when need
+      concurrencyPolicy: "Replace"
+      successfulJobsHistoryLimit: ${{SUCCESS_JOBS_HISTORY_LIMIT}}
+      failedJobsHistoryLimit: ${{FAILED_JOBS_HISTORY_LIMIT}}
+      jobTemplate:
+        metadata:
+          labels:
+            app: ${NAME}-${ZONE}
+            cronjob: ${NAME}-${ZONE}
+        spec:
+          backoffLimit: ${{JOB_BACKOFF_LIMIT}}
+          template:
+            metadata:
+              labels:
+                app: ${NAME}-${ZONE}
+                cronjob: ${NAME}-${ZONE}
+            spec:
+              containers:
+                - name: ${NAME}-${ZONE}-${COMPONENT}-restore
+                  # use the same image as our database, so we can run the psql command
+                  image: image-registry.apps.silver.devops.gov.bc.ca/${OC_NAMESPACE}/${NAME}-${ZONE}-${COMPONENT}:${ZONE}-db
```

Comment:
add the new OC_NAMESPACE variable because we're having problem to pull images before, I got the same error, so pull the image from this url

Reply:
Why are you defining a cron job to test database restore? We don't want this running normally, especially not in production. But perhaps I'm misunderstanding the scope of what this restore process is trying to do. What I'd like to see first before these changes is the documented database restore process (in Confluence). That's what we're trying to test.